benchopt/runner.py

```diff
@@ -497,4 +504,8 @@ def run_benchmark(benchmark, solver_names=None, forced_solvers=None,
     if plot_result:
         from benchopt.plotting import plot_benchmark
         plot_benchmark(save_file, benchmark, html=html)
+
+    if do_profile:
+        print_stats()
```

Comment:
You can simply call this as the Profiler object will only be created if `do_profile` is `True` and at least one function as been decorated.
```suggestion
    print_stats()
```